vlasovsolver/cuda_acc_map_kernel.cu

```diff
@@ -34,7 +55,7 @@ static void HandleError( cudaError_t err, const char *file, int line )
 
 __global__ void acceleration_1
```

Comment:
The naming of both the "caller" and the actual kernel functions should indeed be considered. I guess explicitly calling them "_glue" and "_kernel" would make sense, just so that future people looking at the code aren't completely confused.

Reply:
Good idea!

I'll comment here also on the history of the plm,ppm,pqm files. Yes, that loss of history is unfortunate... but at this point recovering that would sound a bit challenging. If I checkout a file from a previous commit, would that history then get recovered? I'll give it a try.

Reply:
Gave it a few attempts but seems non-trivial. I think we can live with the loss of that history.

Will soon update with renaming and deletion of the wrappercaller.

Reply:
Ok @ursg finally I got the history back, with the caveat that now the renaming couldn't be done (or github again loses the history). But in this PR now the history is back. And the wrappercaller was deleted.